cli.js

```diff
@@ -1,9 +1,19 @@
+require('marko/node-require').install()
+
 const argly = require('argly')
 const mochaPuppeteer = require('./index')
 const _loadConfig = require('./src/utils/loadConfig')
 
 const mochaPuppeteerPkgVersion = require('./package.json').version
 
+function _resolveTestPage (testPagePath) {
+  try {
+    return require(testPagePath)
```

Comment:
This will break if the `testPagePath` is not an absolute path. You should use `resolve-from` to validate the path exists before performing the require. Make the current working directory the basePath.

Reply:
This is a good idea. Will make this change soon.